jupyterhub/Chart.yaml

```
@@ -9,3 +11,6 @@ sources:
 icon: https://jupyter.org/assets/hublogo.svg
 kubeVersion: '>=1.11.0-0'
 tillerVersion: '>=2.11.0-0'
+maintainers:
+  - name: Erik Sundell
+    email: erik@sundellopensource.se
```

```suggestion
    email: erik@sundellopensource.se
  - name: Simon Li
    url: https://github.com/manics/
```
```
maintainers: # (optional)
  - name: The maintainers name (required for each maintainer)
    email: The maintainers email (optional for each maintainer)
    url: A URL for the maintainer (optional for each maintainer)
```
Let me think about which email address to use.....